lib/playground.dart

```diff
@@ -348,6 +352,7 @@ class Playground extends EditorUi implements GistContainer, GistController {
       Channel.fromVersion('stable'),
       Channel.fromVersion('beta'),
       Channel.fromVersion('old'),
+      Channel.fromVersion('dev', hidden: true),
```

Comment:
I'm guessing the actual API URL is defined elsewhere?

Reply:
Yes, the Channel class uses the `devServerUrl` constant from the `common.dart` library.